src/ig/IGExporter.ts

```diff
@@ -75,50 +82,27 @@ export class IGExporter {
   }
 
   /**
-   * Initializes the ImplementationGuide JSON w/ data from the package.json
+   * Initializes the ImplementationGuide JSON w/ data from the configuration YAML
    *
    * @see {@link https://confluence.hl7.org/pages/viewpage.action?pageId=35718629#NPMPackageSpecification-PackageManifestpropertiesforIGs}
    */
-  private initIG(): void {
-    const packageJSON = this.pkg.packageJSON;
-    // TODO: Consider adding a generated file warning when the IG Publisher supports JSON5-style comments
+  private initIG() {
+    // first, properties that can be directly used without much trouble
     this.ig = {
       resourceType: 'ImplementationGuide',
-      id: packageJSON.name,
-      url: `${packageJSON.canonical}/ImplementationGuide/${packageJSON.name}`,
-      version: packageJSON.version,
+      id: this.config.id,
+      url: `${this.config.url}/ImplementationGuide/${this.config.name}`,
```

Comment:
Doesn't the format for the url have to be `<canonical>/<resourceType>/<resource.id>`? So if we are setting `id` to `this.config.id`, don't we need to set `url` to `${this.config.url}/ImplementationGuide/${this.config.id}`?

Reply:
Yes.  Whoops.  I missed that one.  I expect this is just a remnant of the fact that with `packageJSON`, `packageJSON.name` _was_ the `id` of the IG.